src/css/app.scss

```diff
@@ -342,9 +370,19 @@ footer {
     }
 
     @media (max-width: #{map-get($breakpoints, 'md')}) {
+        .footer-content {
+            padding: 0 40px;
+            flex-direction: column;
+            align-items: flex-start;
+
+            .download-error-button {
+                margin-top: 2em;
+            }
```

Comment:
Am I reading this right that we don't have the button on mobile widths?

Reply:
The button is still visible on mobile. I just hid it when we include the email newsletter in the footer (on the home page), since it seems a user wouldn't need to use that button at that point? 🤷‍♀️

Reply:
Ah, yep, sounds right---I think I missed the containing context there :)